app/screens/channel/channel_nav_bar/channel_title/channel_title.js

```diff
@@ -86,14 +93,30 @@ export default class ChannelTitle extends PureComponent {
     }
 
     render() {
-        const {currentChannelName, displayName, isChannelMuted, onPress, theme} = this.props;
+        const {
+            isSelfDMChannel,
+            currentChannelName,
+            displayName,
+            isChannelMuted,
+            onPress,
+            theme,
+        } = this.props;
 
-        const style = getStyle(theme);
+        const {intl} = this.context;
```

Comment:
Are we able to use `intlShape` instead here? I know we're planning on upgrading `react-intl` that will deprecate this functionality.

Reply:
Sorry, I misspoke, still trying to understand it myself lol.
The `contextTypes` method of injecting intl is being removed once we upgrade, so we should be using injectIntl or the FormattedMessage component to do our formatting.

Reply:
Thanks Miguel!